src/NServiceBus.TransactionalSession/ITransactionalSession.cs

```diff
@@ -11,8 +11,8 @@
     public interface ITransactionalSession : IBatchedMessageSession
     {
         /// <summary>
-        /// Gets the synchronized storage session for processing the current message. NServiceBus makes sure the changes made
-        /// via this session will be persisted before the message receive is acknowledged.
+        /// Gets the synchronized storage session for processing the current message.
+        /// Operations performed on the storage session and message operations (send and publish) for a single unit of work and will be atomically committed.  
```

Comment:
```suggestion
        /// Gets the synchronized storage session for processing the current message, which is used to atomically execute any operations performed on the storage session, as well as all the message operations (send and publish)
```

Reply:
>  for processing the current message

I don't understand this part

Reply:
I think having shorter sentences will be easier to read.